src/components/TransactionFailText/index.tsx

```diff
@@ -34,28 +35,31 @@ export const TransactionFailText = ({
   const classes = useStyles()
   const threshold = useSelector(currentSafeThreshold)
   const isWrongChain = useSelector(shouldSwitchWalletChain)
+  const isGranted = useSelector(grantedSelector)
 
   if (txEstimationExecutionStatus !== EstimationStatus.FAILURE) {
     return null
   }
 
-  let errorMessage = 'To save gas costs, avoid creating the transaction.'
+  let errorDesc = 'To save gas costs, avoid creating the transaction.'
   if (isExecution) {
-    errorMessage =
+    errorDesc =
       threshold && threshold > 1
         ? `To save gas costs, reject this transaction`
         : `To save gas costs, avoid executing the transaction.`
   }
 
+  const error = isGranted
+    ? `This transaction will most likely fail. ${errorDesc}`
+    : isWrongChain
+    ? 'Your wallet is connected to the wrong chain.'
+    : "You are currently not an owner of this Safe and won't be able to submit this transaction."
```

Comment:
This is not possible because of the apostrophe in the sentence.

Reply:
You can escape it like so: `'I\'m a JS string'`.
 Or does ESLint mess with our quotes?

Reply:
I escaped it but our formatter converted it to double quotes. I assume it would do the same with backticks as well. I'd be apprehensive to use them everywhere in case of newlines.